nvssclient.tests/Integration/TimedServiceShould.cs

```diff
@@ -27,7 +27,9 @@
 using Microsoft.EntityFrameworkCore;
 using Xunit;
 
-namespace NVSSClient.tests {
+namespace NVSSClient.tests 
+{
+     [Collection("ClientIntegrationTests")]
     public class TimedServiceShould : IClassFixture<CustomWebApplicationFactory<NVSSClient.Startup>>, IDisposable
     {
```

Comment:
There are a few unused variables flagged below that would be good to clean up

Reply:
I'm not sure why it flags _scopeFactory because that is used to define the scope for the database context. Were there any other unused variables?